### ISSUE TITLE
Fix S2479 FP: add support for verbatim interpolated string

### ISSUE THREAD

**Issue by andrei-epure-sonarsource** [SonarSource/sonar-dotnet#2773]
### Description

from [community](https://community.sonarsource.com/t/verbatim-string-with-interpolation-causing-false-positives-with-rule-c-s2479/16026)

We do not support verbatim interpolated strings (C#7 syntax)

### Repro steps

```
// FP below
string testB = $@"asdasdasd
        asasdasd{testA}asdasd
        asdasd";
```

### Related information

* SonarC# Version 8.0
* MSBuild 16

**Comment by andrei-epure-sonarsource** [comment]
also reported by @danbrad in #2839
